jobs/env/ci-kubernetes-e2e-gce-large-performance.env

```diff
@@ -1,5 +1,6 @@
 # Override GCE defaults.
 ALLOWED_NOTREADY_NODES=20
+HEAPSTER_MACHINE_TYPE=n1-standard-8
```

Comment:
Could you also change this in 2 other places:

- jobs/env/ci-kubernetes-e2e-gce-big-performance.env
- jobs/env/ci-kubernetes-e2e-gce-large-manual-up.env